aip_x1_launch/launch/lidar.launch.xml

```diff
@@ -22,6 +22,8 @@
         <arg name="scan_phase" value="180.0" />
         <arg name="launch_driver" value="$(var launch_driver)" />
         <arg name="vehicle_mirror_param_file" value="$(var vehicle_mirror_param_file)" />
+        <arg name="use_pointcloud_container" value="$(var use_pointcloud_container)" />
+        <arg name="container_name" value="$(var pointcloud_container_name)"/>
```

Comment:
To Ota-san
x1 is no longer using this repository, so it shouldn't affect your project, but please read it for reference.